xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/SaveAction.java

```diff
@@ -54,8 +61,15 @@
     /** The identifier of the save action. */
     public static final String ACTION_NAME = "save";
 
+    /**
+     * The key to retrieve the saved object version from the context.
+     */
+    public static final String SAVED_OBJECT_VERSION_KEY = "SaveAction.savedObjectVersion";
```

Comment:
Not super fan of this hack being an API (which this new static is). At the very least it should stay private for now (in which case it's a little better because we can refactor it easily later).

Reply:
It cannot be private since I need it in SaveActionAndContinue.

Reply:
You don't really need that field, you need each code to use the same key name. I really prefer duplicate over new API for this one. You could also move it in some internal class.

Reply:
OK, I'll duplicate it then.